ingestion/src/metadata/profiler/metrics/static/max.py

```diff
@@ -40,6 +40,12 @@ def _(element, compiler, **kw):
     return f"MAX({col})"
 
 
+@compiles(MaxFn, Dialects.Trino)
+def _(element, compiler, **kw):
+    col = compiler.process(element.clauses, **kw)
+    return f"IF(is_nan(MAX({col})), NULL, MAX({col}))"
+
+
```

Comment:
Checks if_nan, send None instead of Null

Reply:
Do you know in what scenario this fails? The query below against a column with NaN worked for me

```
select MAX(average_time)
FROM "system".runtime.optimizer_rule_stats;
```

Reply:
this fails when the column has NaN in it, removing this gives us
`value is nan` failure